src/Entity/UserFileAction.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\UserFileActionRepository;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: UserFileActionRepository::class)]
+class UserFileAction
```

Comment:
Jakie będzie przeznaczenie tej encji? Czy to będzie obiekt przechowujący akcje na plikach wykonane przez użytkownika np. `upload`, `download` lub `remove`?

Reply:
tak, to jest encja służąca nam do przechowywania historii działań user'a / admina np.

Reply:
dobra zlącze to bo ruszamy xd ale next sprawdzamy d